content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+and an embedded video demonstration of this process.
+
+== YAML File Location
+
+By default, the YAML file for the CasC configuration
+is located in `$JENKINS_HOME/jenkins.yaml`.
+The location and name of the file being used is displayed
+on the *Configuration as Code* UI page.
+You can specify a different file to view by typing the full pathname
+into the *Path or URL* field.
+
+You can specify a different location or a different file name
+for the creation of the JCasC YAML file by doing either of the following:
+
+* Populate the `CASC_JENKINS_CONFIG` environment variable to point to
+a comma-separated list that defines where configuration files are located.
+
+* Use the `casc.jenkins.config` Java property to control the file name and location.
+This is useful when installing Jenkins via a package management tool.
+It can not set an environment variable outside of a package-managed file,
+because it could be overwritten by an update.
++
+
+On RHEL/CentOS systems, you can append the following to the
+`JENKINS_JAVA_OPTIONS` entry in the `/etc/sysconfig/jenkins` file:
++
+[source,bash]
+----
+`-Dcasc.jenkins.config=/jenkins/casc_configs`
+----
+
+The file location and name can be specified as any of the following:
+
+* Path to a folder containing a set of config files such as `/var/jenkins_home/casc_configs`.
+* A full path to a single file such as `/var/jenkins_home/casc_configs/jenkins.yaml`.
+* A URL pointing to a file served on the web such as `https://acme.org/jenkins.yaml`.
+
+The value of the `CASC_JENKINS_CONFIG` variable is unpacked
+according to the following rules:
+
+* If an element of `CASC_JENKINS_CONFIG` points to a folder,
+the plugin recursively traverses the folder to find file(s)
+with the .yml, .yaml, .YAML, or .YML suffix.
+
+* It excludes hidden files or files that contain a hidden folder
+(such as/ `jenkins/casc_configs/.dir1/config.yaml`)
+in **any part** of the full path.
+
+* It follows symbolic links for both files and directories.
+
+* The order of traversal does not matter to the final outcome
+because all configuration files that are discovered MUST be supplementary.
+If a file attempts to overwrite configuration values from another file,
+it creates a conflict and raises a `ConfiguratorException`.
+
+=== CasC Configuration and UI Modifications
+
+Configuration for a Jenkins controller should be implemented
+either with CasC or with the UI, but not by both.
+By default, the system allows you to use the UI
+to modify configuration options that were configured by CasC,
+but these modifications are overwritten the next time the controller restarts.
+
+To ensure that configuration changes are not made through the UI,
+install and enable the
+link:https://www.jenkins.io/blog/2020/05/25/read-only-jenkins-announcement/[Read-only Jenkins Configuration]
+plugin.
+When this plugin is installed,
+all configuration fields set by JCasC display as read-only values
+that cannot be modified through the UI.
+
+== For More Information
```

Comment:
The [style guide](https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc#assorted-comments) requests sentence case the "titles" (I interpret that to mean "headings", but that is not a consistent usage throughout the documents, for sure) (optional)

```suggestion
== For more information
```